app/services/food_service.rb

```diff
@@ -0,0 +1,18 @@
+class FoodService
+  def self.call_for_food(food)
+    response = conn.get("/fdc/v1/foods/search?api_key=#{ENV['PARKS_API_KEY']}&query=#{food}&Size=10&brandOwner")
```

Comment:
I don't think you'll need `brandOwner` at the end of this endpoint

Reply:
Looks like you may have referenced a different practice when making this service which is totally fine... Take a look at your conn method below. In there, you can pass the api key as a parameter. The docs for this api does mention that the api will be passed as a param instead of a header, so for that, you'll just need to change `faraday.headers` to `faraday.param`.